HTML/mockup-minimo/index.html

```diff
@@ -0,0 +1,185 @@
+                    <source media="(min-width: 500px)" srcset="img/image_2@2x-min.png">
+                    <img src="img/image_2-min.png" class="article__image" alt="article photo">
+                </picture>
+                <h3 class="article__tag">lifestyle</h3>
+                <h2 class="article__header">Life tastes better with coffee
+                </h2>
+                <p class="article__paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
+                    et
+                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                </p>
+            </article>
+            <article>
+                <picture>
+                    <source media="(min-width: 1200px)" srcset="img/image_3@3x-min.png">
+                    <source media="(min-width: 500px)" srcset="img/image_3@2x-min.png">
+                    <img src="img/image_3-min.png" class="article__image" alt="article photo">
+                </picture>
+                <h3 class="article__tag">photodiary</h3>
+                <h2 class="article__header">American dream
+                </h2>
+                <p class="article__paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
+                    et
+                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                </p>
+            </article>
+            <article>
+                <picture>
+                    <source media="(min-width: 1200px)" srcset="img/image_4@3x-min.png">
+                    <source media="(min-width: 500px)" srcset="img/image_4@2x-min.png">
+                    <img src="img/image_4-min.png" class="article__image" alt="article photo">
+                </picture>
+                <h3 class="article__tag">photodiary</h3>
+                <h2 class="article__header">A day exploring the Alps
+                </h2>
+                <p class="article__paragraph">
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore
+                    et
+                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                </p>
+            </article>
+        </section>
+        <section class="section-sign-in">
+            <h2 class="section-sign-in__header article__header">Sign up for our newsletter!</h2>
+            <form class="section-sign-in__form">
+                <input type="email" class="section-sign-in__input" placeholder="Enter a valid email address">
+                <input type="submit" class="section-sign-in__button" value=&rarr;>
+            </form>
+        </section>
+        <section class="section-posts">
+            <div class="section-posts--second">
```

Comment:
you can't use modifier without base class which behaviour you modify, maybe you can remove here this `div` not sure about it